src/functionalTest/resources/environment.conf

```diff
@@ -9,7 +9,8 @@ serviceBus {
   namespaceConnectionString = ${TEST_SERVICE_BUS_CONNECTION_STRING}
   topic = ${TEST_SERVICE_BUS_TOPIC}
   subscription = ${TEST_SERVICE_BUS_SUBSCRIPTION}
-  pollingDelayInMs = ${TEST_SERVICE_BUS_POLLING_DELAY_MS}
-  messageLockTimeoutInMs = ${TEST_SERVICE_BUS_MESSAGE_LOCK_TIMEOUT_MS}
+  //  todo: retrieve from env var, set based on servicebus config in tf
+  pollingDelayInMs = 30000
+  messageLockTimeoutInMs = 60000
```

Comment:
lock duration cannot be currently retrieved from the subscription tf module.
Using this default value for now.

see https://github.com/hmcts/terraform-module-servicebus-subscription/blob/master/variables.tf#L30